tools/config/go-templates/README.mustache

```diff
@@ -4,9 +4,9 @@
 
 All URIs are relative to *{{basePath}}*
 
-Class | Method | HTTP request | Description
------------- | ------------- | ------------- | -------------
-{{#apiInfo}}{{#apis}}{{#operations}}{{#operation}}*{{classname}}* | [{{operationId}}](./{{apiDocPath}}{{classname}}.md#{{operationIdLowerCase}}) | **{{httpMethod}}** {{path}} | {{summary}}
+Class        | Method        | HTTP request  | Description   | Experimental   
```

Comment:
like discussed on the call can we make the tittle of the table, "Stability level" and instead of yes no, make it Stable/Experimental?

Reply:
Done

Reply:
Experimental means = we have that untested in SDK. "Stability level" is not documented term and likely to mean API stability levels in the future. We can imagine having alpha stability level being experimental and also Production also being untested.